circle.yml

```diff
@@ -0,0 +1,15 @@
+machine:
```

Comment:
I don't have a preference for CI, I just know that Travis is very popular, so I was curious about the differences.